assets/app/scripts/controllers/events.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:EventsController
+ * @description
+ * # EventsController
+ * Controller of the openshiftConsole
+ */
+angular.module('openshiftConsole')
+  .controller('EventsController', function ($scope, DataService, Logger) {
+    $scope.events = {};
+    var watches = [];
+
+    watches.push(DataService.watch("events", $scope, function(events) {
+      $scope.events = events.by("metadata.name");
```

Comment:
do events have labels that would be useful to allow you to filter on? if so, let's hook that up. If not, make sure the label filter widget doesn't show